crates/runc/Cargo.toml

```diff
@@ -26,6 +26,7 @@ tempfile = "3.3.0"
 thiserror = "1.0.30"
 time = { version = "0.3.7", features = ["serde", "std"] }
 uuid = { version = "0.8.2", features = ["v4"] }
+containerd-shim = { path = "../shim", version = "0.2.0" }
```

Comment:
`runc` crate expected to be an independent from containerd CLI wrapper for runc binary. We should not depend on containerd-shim crate here.

Reply:
maybe  we can change the runc crate to an "oci" crate, it's just a wrapper of runc binary call now, and runc is a standard implement of OCI runtime spec. we can change the command path from hard coded "runc" to an abstract method, say "get_command_path()", other oci compatible runtimes can easily use this crate to make their own shim-v2 binary, by implementing get_command_path()

Reply:
> `runc` crate expected to be an independent from containerd CLI wrapper for runc binary. We should not depend on containerd-shim crate here.

`runc` crate needs IO and we take IO as a common trait for various runtime in `shim` crate, that's why it imports shim. I can follow your mind to put IO in runc as you always do. 
Could you wait for my new submit around this Fri possiably, otherwise I cam push in the followup PR.

Reply:
Sure, no rush, can wait till Friday.